### ISSUE TITLE
Big Picture / Steam Deck mode

### ISSUE THREAD

**Issue by cheater** [lutris/lutris#4466]
This has been something people have [wanted](https://www.reddit.com/r/linux_gaming/comments/drfvh8/is_there_a_big_mode_front_end_for_lutris/) for a [few](https://www.reddit.com/r/Lutris/comments/opkywx/is_there_a_controller_navigable_version_of_lutris/) years now, and it would be a serious QoL improvement on the Steam Deck. I understand it's a [patreon goal](https://www.patreon.com/lutris) and it's very far from being reached. That's fine, I'm just here to record the request so people know it's in some way on the map (even if not planned on being worked on right now) and what they can do to help it along. I've looked through open issues and it's not in there.

**Comment by jclsn** [comment]
Yeah, it would be nice to have an interface like in Steam, but I doubt it will ever reach the quality that the Steam Big Picture mode offers. I think as a first step the adding of games to the Steam library should be improved. As of now, you can add it and then add the missing grids and cover arts with [Steam Boop](https://www.steamgriddb.com/boop), but this doesn't always work. It would be great if this could be fully automated by Lutris. 

On the other hand this would make Lutris merely an organization tool and not an interface people would use. Having their own Big Picture mode would also be nice, especially because it would be lacking all the commercial ads etc. Much like Plex and Jellyfin. I prefer Jellyfin, because it is less bloated.

**Comment by strycore** [comment]
There are plans to implement a full screen UI using Godot. This hasn't been a priority so far because 1) Godot 4 wasn't released until recently and 2) we wanted to focus on getting Lutris games to work on the Steam Deck's game UI since that's the better option on the Deck.
